### PR TITLE
Hotfix/wording choice

### PR DESCRIPTION
**Pull request by mxng**
## Why

Some phrases are confusing or vague

## What

The wordings have changed

### CODE REVIEW

**Comment by mxng** [comment]
[move up the "be the first to review"](https://trello.com/c/T1MsMV97/209-move-up-the-be-the-first-to-review)

File: app/views/responses/show.html.erb [hunk 1]
```diff
@@ -35,19 +35,18 @@
...
       <div class="modal-header">
-        <h5 class="modal-title" id="exampleModalCenterTitle">Generate Results</h5>
+        <h5 class="modal-title" id="exampleModalCenterTitle">Show me where to eat</h5>
         <button type="button" class="close" data-dismiss="modal" aria-label="Close">
           <span aria-hidden="true">&times;</span>
         </button>
       </div>
       <div class="modal-body">
-        Are you sure that you want to end the poll prematurely and generate your restaurant result?
+        Are you sure that you want to end the poll prematurely and see your restaurant result?
```
Comment (TeeWhyKay) [inline app/views/responses/show.html.erb:49]: End the poll now and show me where to eat!

Comment (TeeWhyKay) [reply app/views/responses/show.html.erb:49]: Button can be Yes / No (to cancel)

**Review by TeeWhyKay — APPROVED** [review]
Other than the comment i made, looks fine so far.

File: app/views/responses/show.html.erb (hunk 1 above)
Comment (mxng) [reply app/views/responses/show.html.erb:49]: > Button can be Yes / No (to cancel)

👌

Comment (mxng) [reply app/views/responses/show.html.erb:49]: > End the poll now and show me where to eat!
localhost_3000_chomp_sessions_944532_responses_3(iPhone 6_7_8)

This ok?

**Review by amelialimyh — APPROVED** [review]
Looks alright

**Comment by mxng** [comment]
Please go through the flow and give feedback, and check for consistency, thanks!!

**Pull request merged**